src/porepy/grids/mortar_grid.py

```diff
@@ -78,6 +78,9 @@ def __init__(self, dim, side_grids, face_cells, name=""):
         self.cell_volumes = np.hstack(
             [g.cell_volumes for g in self.side_grids.values()]
         )
+        self.cell_centers = np.hstack(
```

Comment:
Are we introducing more of a geometry for the mortar grid? I see no problems right now, and this is probably a reasonable approach, but I want to understand what is going on.

Reply:
OK, I'll put a comment. Also, I'm not tracking this attribute for changing in the mortar grid. I should modify the corresponding tests to do it..

Reply:
no need for the test after a careful check